src/icon-tab-bar.tsx

```diff
@@ -20,37 +21,34 @@ type Props = {
 export const IconTabBar: FC<Props> = ({ tabs, selectedTab, onTabSelected }) => {
   const iconTabs = tabs.map(project => {
     const isSelected = project === selectedTab;
-    const buttonClasses = [
-      'min-w-24',
-      'h-12',
-      'border',
-      'rounded-3xl',
-      'px-4',
-      'py-3',
-      'flex',
-      'gap-2',
-      'items-center',
-      'justify-center',
-    ];
-    if (isSelected) {
-      buttonClasses.push(
-        'cursor-default',
-        'text-white',
-        'bg-purple-500',
-        'border-purple-500',
-      );
-    } else {
-      buttonClasses.push(
-        'cursor-pointer',
-        'text-purple-500',
-        'bg-transparent',
-        'border-grey-300',
-      );
-    }
     return (
       <button
         key={project}
-        className={buttonClasses.join(' ')}
+        className={clsx(
+          'min-w-24',
+          'h-12',
+          'border',
+          'rounded-3xl',
+          'px-4',
+          'py-3',
+          'flex',
+          'gap-2',
+          'items-center',
+          'justify-center',
+          isSelected
+            ? [
+                'cursor-default',
+                'text-white',
+                'bg-purple-500',
+                'border-purple-500',
+              ]
+            : [
+                'cursor-pointer',
+                'text-purple-500',
+                'bg-transparent',
+                'border-grey-300',
+              ],
```

Comment:
this is a bit of a nit, but I think I'd prefer if we didn't use a ternary for classnames; `clsx` doesn't list it as one of the options in their [Usage](https://github.com/lukeed/clsx?tab=readme-ov-file#usage) examples, which is probably not a big deal but does imply a potential anti-pattern. I also think it's a bit unwieldy compared to something like

```
{
  'cursor-default text-white bg-purple-500 border-purple-500': isSelected,
  'cursor-pointer text-purple-500 bg-transparent border-grey-300': !isSelected
}
```

Reply:
I switched to a different pattern from the docs instead: the `condition && 'string'` style. This is specifically mentioned in their section on Tailwind, and offers the ability to switch to `clsx/lite` if we want. It also reads clearer to me than the object style, by leading with the condition rather than the classes. How do you feel about this?

Reply:
love it 🤝